### DISCUSSION TITLE
embedding JBrowse2

### DISCUSSION THREAD

**Discussion by vpandey-om** [GMOD/jbrowse-components#2150] [Q&A]
Dear Developers, 
I used this tutorial (https://jbrowse.org/jb2/docs/bcc2020_embedding_jbrowse_08_default_session) to embed JBrowse2  in my flask app.

For example, 

aliases: ['hg38'],
  refNameAliases: {
    adapter: {
      type: 'RefNameAliasAdapter',
      location: {
        uri: 'https://s3.amazonaws.com/jbrowse.org/genomes/GRCh38/hg38_aliases.txt', (**Working** )
        uri: 'hg38_aliases.txt' ((**Not Working  Error is : Failed to load resource: the server responded with a status of 404 
    (NOT FOUND)**)), 
      },
    },
  },

I am very new in web development and was wondering how to serve files locally.

I was wondering if you have any solutions. 

Best
Vikash

**Comment by garrettjstevens** [comment]
I'm not familiar with Flask, but what you will need is static file hosting. If you look for a guide that is something like "How to serve static files in Flask," that should point you in the right direction.
